tfhe/docs/getting_started/operations.md

```diff
@@ -1,105 +1,118 @@
+| Greater or equal than | `ge`           | :heavy_check_mark: | :heavy_check_mark:       |
+| Lower than            | `lt`           | :heavy_check_mark: | :heavy_check_mark:       |
+| Lower or equal than   | `le`           | :heavy_check_mark: | :heavy_check_mark:       |
+| Equal                 | `eq`           | :heavy_check_mark: | :heavy_check_mark:       |
+| Cast (into dest type) | `cast_into`    | :heavy_check_mark: | :heavy_multiplication_x: |
+| Cast (from src type)  | `cast_from`    | :heavy_check_mark: | :heavy_multiplication_x: |
+| Ternary operator      | `if_then_else` | :heavy_check_mark: | :heavy_multiplication_x: |
 
-In Rust, operations on native types are modular. For example, computations on `u8` are carried out modulo $$2^{8}$$. A similar idea applies for FheUintX, where operations are done modulo $$2^{X}$$. For FheUint3, operations are done modulo $$8 = 2^{3}$$.
 
-### Arithmetic operations.
+## Integer
 
-Small homomorphic integer types support all common arithmetic operations, meaning `+`, `-`, `x`, `/`, `mod`.
+In `TFHE-rs`, integers are used to encrypt any messages which are larger than 4 bits. All supported operations are listed below.
 
-The division operation implements a subtlety: since data is encrypted, it is possible to compute a division by 0. In this case, the division is tweaked so that dividing by 0 returns the max possible value for the message.
+### Arithmetic operations.
+
+Homomorphic integer types support arithmetic operations.
 
 The list of supported operations is:
 
-| name                                                    | symbol | type   |
-| ------------------------------------------------------- | ------ | ------ |
-| [Add](https://doc.rust-lang.org/std/ops/trait.Add.html) | `+`    | Binary |
-| [Sub](https://doc.rust-lang.org/std/ops/trait.Sub.html) | `-`    | Binary |
-| [Mul](https://doc.rust-lang.org/std/ops/trait.Mul.html) | `*`    | Binary |
-| [Div](https://doc.rust-lang.org/std/ops/trait.Div.html) | `/`    | Binary |
-| [Rem](https://doc.rust-lang.org/std/ops/trait.Rem.html) | `%`    | Binary |
-| [Neg](https://doc.rust-lang.org/std/ops/trait.Neg.html) | `-`    | Unary  |
+| name                                                     | symbol | type   |
+|----------------------------------------------------------|--------|--------|
+| [Neg](https://doc.rust-lang.org/std/ops/trait.Neg.html)  | `-`    | Unary  |
+| [Add](https://doc.rust-lang.org/std/ops/trait.Add.html)  | `+`    | Binary |
+| [Sub](https://doc.rust-lang.org/std/ops/trait.Sub.html)  | `-`    | Binary |
+| [Mul](https://doc.rust-lang.org/std/ops/trait.Mul.html)  | `*`    | Binary |
+| [Div](https://doc.rust-lang.org/std/ops/trait.Div.html)* | `/`    | Binary |
+| [Rem](https://doc.rust-lang.org/std/ops/trait.Rem.html)* | `%`    | Binary |
+
+For division by 0, the convention is to return modulus - 1. For instance, for `FheUint8`, the modulus is $$2^8=256$$, so a division by 0 will return an encryption of 255.
```

Comment:
Not sure if this is inconsistent, so adding a comment instead of making a change. Maybe 

modulus - 1 to (modulus - 1) or `modulus -1`?